R/github-labels.R

```diff
@@ -22,8 +22,8 @@
 #' * `good first issue` indicates a good issue for first-time contributors.
 #' * `help wanted` indicates that a maintainer wants help on an issue.
 #'
-#' @param repo Optional repository specification (`owner/repo`) if you
-#'   don't want to use the current project.
+#' @param repo_spec Optional repository specification (`owner/repo`) if you
```

Comment:
For consistency with `create_from_github()`, `use_tidy_thanks()`, and lots of internal usage.